### PR TITLE
Ds/quad nodes

### PR DESCRIPTION
**Pull request by dsteinmo**
Happy 34th birthday to me -- quads!

### CODE REVIEW

File: include/LinAlgHelpers.hpp [hunk 1]
```diff
@@ -96,4 +96,14 @@ namespace blitzdg {
     inline real_type normFro(const matrix_type<T>& mat) {
         return std::sqrt(blitz::sum(mat * mat));
     }
+
+    /**
+     * Helper method for checking that two points (x1,y1) and (x2,y2) are less than a distance eps apart.
+     */
+    inline bool distanceLessThanEps(real_type x1, real_type y1, real_type x2, real_type y2, real_type eps) {
```
Comment (k7miller) [inline include/LinAlgHelpers.hpp:103]: I would write this as
```
return std::hypot(x2 - x1, y2 - y1) < eps;
```

File: include/LinAlgHelpers.hpp [hunk 2]
```diff
@@ -96,4 +96,14 @@ namespace blitzdg {
     inline real_type normFro(const matrix_type<T>& mat) {
         return std::sqrt(blitz::sum(mat * mat));
     }
+
```
Comment (k7miller) [inline include/LinAlgHelpers.hpp:99]: Not sure about your implementation of the Frobenius norm above. It depends on what type of product the `*` operator implements. If its elementwise multiplication then you are good, but if its matrix-matrix multiplication then you want the trace of `mat * mat`.

**Review by k7miller — COMMENTED** [review]
That's one hell of a large commit Frank. Looks pretty good! Nice work.

File: include/LinAlgHelpers.hpp (hunk 2 above)
Comment (dsteinmo) [reply include/LinAlgHelpers.hpp:99]: matrix-matrix multiplication is written as a blitz summation over the inner index, mat * mat is the defacto element-wise product. Thanks for pointing this out as a potential point of future confusion though...

File: include/LinAlgHelpers.hpp (hunk 1 above)
Comment (dsteinmo) [reply include/LinAlgHelpers.hpp:103]: cool, will do.

**Pull request merged**